integration_test.go

```diff
@@ -55,6 +59,40 @@ func testCartographerMap(t *testing.T, svc slam.Service, localizationMode bool)
 	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
 }
 
+func testCartographerFullModPosition(t *testing.T, svc slam.Service, expectedComponentRef string) {
+	expectedPosOSX := r3.Vector{X: -1.0283431004415964, Y: 1.9804921951032644, Z: 0}
+
+	expectedPosLinux := r3.Vector{X: -0.062249046977189776, Y: 2.97876740701877, Z: 0}
+	tolerancePos := 0.001
+	expectedOri := &spatialmath.R4AA{Theta: 0, RX: 0, RY: 0, RZ: 1}
+	toleranceOri := 0.001
+
+	position, componentRef, err := svc.GetPosition(context.Background())
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, componentRef, test.ShouldEqual, expectedComponentRef)
+
+	actualPos := position.Point()
+	t.Logf("Position point: (%v, %v, %v)", actualPos.X, actualPos.Y, actualPos.Z)
+	// mac
+	if actualPos.X > expectedPosOSX.X-tolerancePos && actualPos.X < expectedPosOSX.X+tolerancePos {
```

Comment:
This proves that cartographer is deterministic (at least with in a sub-millimeter margin of error) on mac & linux but gets different results for some reason between mac & Linux.